src/tasks/removeBirthdayRole.ts

```diff
@@ -0,0 +1,23 @@
+import { PartialResponseValue, ResponseType, Task } from '#lib/database';
+
+export class UserTask extends Task {
+	public async run(data: RemoveBirthdayRoleData): Promise<PartialResponseValue | null> {
+		const guild = this.context.client.guilds.cache.get(data.guildID);
+		if (!guild) return null;
+		const member = await guild.members.fetch(data.userID);
+		if (!member) return null;
+
+		const { me } = guild;
```

Comment:
Never going to be null:
```suggestion
		const me = guild.me!;
```